x-pack/plugins/security_solution/common/endpoint/service/authz/authz.test.ts

```diff
@@ -42,6 +43,14 @@ describe('Endpoint Authz service', () => {
         );
       });
 
+      it('should set `canKillProcess` to false if not proper license', () => {
```

Comment:
Could you add a test where we check that it is set to true when proper license?

Reply:
this is done on line 31 above